site/content/en/docs/Concepts/accounts.md

```diff
@@ -46,23 +46,70 @@ or workflows run.
 
 Activating an account will also bootstrap the AWS Environment for CDK app deployments.
 
+#### Using an Existing S3 Bucket
+
 Amazon Genomics CLI requires an S3 bucket to store workflow results and associated information. If you prefer to use an existing bucket
 you can use the form  `agc account activate --bucket my-existing-bucket`. If you do this the AWS [IAM](https://docs.aws.amazon.com/IAM/latest/UserGuide/index.html) role used to run
 Amazon Genomics CLI must be able to write to that bucket.
 
+#### Using an Existing VPC
+
 To use an existing VPC you can use the form  `agc account activate --vpc my-existing-vpc-id`. This VPC must have at least
 3 availability zones each with at least one private subnet. The private subnets must have connectivity to the internet, 
 such as via a NAT gateway, and connectivity to AWS services either through VPC endpoints or the internet. Amazon Genomics CLI will not
 modify the network topology of the specified VPC.
 
+#### Specifying Subnets
+
+When using an existing VPC you may need to specify which subnets of the VPC can be used for infrastructure. This is useful
+when only some private subnets have internet routing. To do this you can supply a comma separated list of subnet IDs using
+the `--subnets` flag, or repeat the flag multiple times. For example:
+
+`agc account activate --vpc my-existing-vpc-id --subnets subnet-id-1,subnet-id-2 --subnets subnet-id-3`
+
+We recommend a minimum of 3 subnets across availability zones to take advantage of EC2 instance availability and to
+ensure high availability of infrastructure.
+
+#### Using Only Public Subnets
+
+Amazon Genomics CLI can create a new VPC with only public subnets to use for its infrastructure using the `--usePublicSubnets` flag.
+
+`agc account activate --usePublicSubnets`
```

Comment:
Please cover use case when a user activates an account with this flag, and then, re-activates it without this flag. And another way around.